src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -258,6 +258,10 @@ def _flexible_server_params(command_group):
                            help='Server major version.')
                 c.argument('zone', options_list=['--zone, -z'],
                            help='Availability zone into which to provision the resource.')
+                c.argument('iops', type=int, options_list=['--iops'],
+                           help='Number of IOPS to be allocated for this server. You will get certain amount of free IOPS based '
```

Comment:
Is there an allowed value set for the parameter?

Reply:
No, it can be any int value.